engine/src/main/java/org/hibernate/search/indexes/impl/IndexManagerHolder.java

```diff
@@ -65,6 +65,10 @@
 
 	private static final Similarity DEFAULT_SIMILARITY = new ClassicSimilarity();
 
+	private static final String DEFAULT_INDEX_MANAGER_KEY = "__DEFAULT__";
+
+	private final Map<String, Class<? extends IndexManager>> indexManagerImplementationsRegistry
```

Comment:
Can be declared as `ConcurrentMap`, making it easier for the reader to reason about the thread-safety (same below).